src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -151,6 +206,57 @@ const FilterableWorkflowTable = ({
     }
   };
 
+  const includeKeys = useMemo(
+    () => [
```

Comment:
I don't think we need any of these keys for the submission details page, this is all for call-level data and we just need a few workflow-level fields. According to [these docs](https://cromwell.readthedocs.io/en/stable/api/RESTAPI/#get-workflow-and-call-level-metadata-for-a-specified-workflow) we should be able to pass `excludeKeys=['calls']` to get just the top-level data, then provide a list of just the top-level fields that we need as `includeKeys`.

For context, this is currently requesting the full dump of metadata that we show on the run details page. That can be huge, and one advantage of offering users these logs here is that we don't need to load that large amount of data.